java/ql/src/semmle/code/java/frameworks/play/PlayBodyParser.qll

```diff
@@ -0,0 +1,11 @@
+import java
+
+/**
+ * Play Framework Explicit Body Parser
+ *
+ * @description Gets the methods using the explicit body parser annotation. The methods are usually controller action methods
+ * (https://www.playframework.com/documentation/2.8.x/JavaBodyParsers#Choosing-an-explicit-body-parser)
+ */
+class PlayBodyParserAnnotation extends Annotation {
```

Comment:
Unused class?

Reply:
The idea was to find methods to see the type of BodyParser the resource calls, like the one's with xml or raw buffer might be interesting to see where it could flow to.